doc/config.md

```diff
@@ -0,0 +1,129 @@
+
+Example:
+
+```yaml
+ aggregate:
+  FLAVOR: Shiny-New-Flavour
+  archs:
+    - aarch64
+    - s390x
+  test_issues:
+    BASE_TEST_ISSUES: Basesystem:17-SP5
+    OS_TEST_ISSUES: Product-Linux:17-SP5
+    SERVERAPP_TEST_ISSUES: Module-Server-Applications:17-SP5
+    DESKTOP_TEST_ISSUES: Module-Desktop-Applications:17-SP5
+    SDK_TEST_ISSUES: Development-Tools:17-SP5
+    PCM_TEST_ISSUES: Public-Cloud:17-SP5
+  onetime: true
+```
+
+* `aggregate` - self-explanatory, mapping containing all needed settings for the aggregate schedule of a given *product*
+* `FLAVOR` - flavour used by the schedule, can be only once per product, mandatory
+* `archs` - list scheduled architectures, mandatory
+* `test_issues` - mandatory, contains a mapping of `*_TEST_ISSUES`, which are used to decide which Incidents to be scheduled in aggregate jobs. Values are `PRODUCT-IDENTIFICATION:VERSION`. `PRODUCT-IDENTIFICATION` as defined in OBS/IBS. `VERSION` dtto, could use different versions for different `*_TEST_ISSUES`.
+  * `OS_TEST_ISSUES` variable is implicit, always used by *os-autoinst-distri-opensuse*, contain identification of the base product.
+  * All others contain modules, addons, and extensions used in this aggregate. First part of the key name must be the same (uppercase, os-autoinst-distri-opensuse will convert it to lowercase) as an addon, extension or module identification in `SCC_ADDONS` variable defined in the job template inside the openQA instance.
+* `onetime` - optional key, boolean. By default, qem-bot sets it to `False`. When set to `True`, it limits bot scheduling this aggregate to only once per day.
+
+## Incidents part of the config
+
+Used to schedule jobs per one Incident. Scheduled jobs will have BUILD **:INCIDENT_NR:shortest_package_name**. All needed links to scheduled incident repositories are in the `INCIDENT_REPO` variable.
+
+Example:
+
+```yaml
+incidents:
+  FLAVOR:
+    Some-Incident-Flavor:
+      archs:
+        - x86_64
+        - aarch64
+      issues:
+         BASE_TEST_ISSUES: Basesystem:17-SP5
+         OS_TEST_ISSUES: Product-Linux:17-SP5
+         SERVERAPP_TEST_ISSUES: Module-Server-Applications:17-SP5
+         DESKTOP_TEST_ISSUES: Module-Desktop-Applications:17-SP5
+         SDK_TEST_ISSUES: Development-Tools:17-SP5
+         PCM_TEST_ISSUES: Public-Cloud:17-SP5
+      required_issues:
+         - PCM_TEST_ISSUES
+         - BASE_TEST_ISSUES
+      aggregate_job: false
+      aggregate_check_true:
+         - KEYONE
+      aggregate_check_false:
+         - KEYTWO
+         - KEYTHREE
+      override_priority: 100
+      params_expand:
+        FOO: foo
+        BAR: bar
+      packages:
+        - pkgone
+        - pkgtwo
+      excluded_packages:
+        - pkgthree
+        - pkgfour
+    Other-Incident-FLavor:
+      ...
+```
+
+* `incidents` mapping contains all settings for an incident schedule of a product.
```

Comment:
```suggestion
* `incidents` mapping containing all settings for an incident schedule of a product.
```

Reply:
This wasn't addressed.
Here you use `mapping contains`, and directly below you use `mapping containing`.